activerecord/test/cases/associations/inverse_associations_test.rb

```diff
@@ -336,15 +336,13 @@ def test_trying_to_use_inverses_that_dont_exist_should_raise_an_error
     assert_raise(ActiveRecord::InverseOfAssociationNotFoundError) { Human.first.confused_face }
   end
 
-  if defined?(DidYouMean) && DidYouMean.respond_to?(:correct_error)
-    def test_trying_to_use_inverses_that_dont_exist_should_have_suggestions_for_fix
-      error = assert_raise(ActiveRecord::InverseOfAssociationNotFoundError) {
-        Human.first.confused_face
-      }
-
-      assert_match "Did you mean?", error.message
-      assert_equal "super_human", error.corrections.first
-    end
+  def test_trying_to_use_inverses_that_dont_exist_should_have_suggestions_for_fix
+    error = assert_raise(ActiveRecord::InverseOfAssociationNotFoundError) {
+      Human.first.confused_face
+    }
+
+    assert_match "Did you mean?", error.message
```

Comment:
Can you take a look at the CI failures?
https://buildkite.com/rails/rails/builds/78906#026004b5-3d08-4211-92cf-06bd328733f2/1086-1095
https://buildkite.com/rails/rails/builds/78906#026004b5-3d08-4211-92cf-06bd328733f2/1086-1104

Reply:
@kamipo, sorry. Should be fixed now.